GOLANG_VERSION

```diff
@@ -1 +1 @@
-1.14.4
+1.16.9
```

Comment:
Go to 1.16.10 instead. There's no reason _not_ to consume later patches.

Reply:
```suggestion
1.16.10
```